plugins/pulp_rpm/plugins/importers/yum/parse/rpm.py

```diff
@@ -120,6 +121,27 @@ def package_headers(filename):
     return headers
 
 
+def package_headers_drpm(filename):
```

Comment:
any reason why you implemented a specific method for drpm header extraction? i guess we do already have one method and it seems likt it is doing the correct thing for rpm,drpm and srpm.